res/values-es/strings.xml

```diff
@@ -992,7 +992,7 @@ Si ves a un jugador que esté rompiendo las reglas del juego, favor de denunciar
     <string name="Name_Opacity_">Opacidad del Nombre:</string>
     <string name="SET_PROFILE_PIC">ESTABLECER FOTO DE PERFIL</string>
     <string name="ACCEPT_ALL_FRIEND_INVITATIONS_">¡ACEPTAR TODAS LAS SOLICITUDES DE AMIGOS!</string>
-    <string name="App_Perf_Stats">Estadísticas Rendimiento App</string>
+    <string name="App_Perf_Stats">Estadísticas de Rendimiento App</string>
```

Comment:
Why not "de rendimiento **del** app?"

Reply:
Ese "de" es para referirse a algo, sin el "de" no sería correcto

Reply:
I am not talking about the "de", but the missing "del" between **rendimiento** and **app**.

Reply:
Corregido